config/src/config/test_config.rs

```diff
@@ -21,6 +21,18 @@ pub struct TestConfig {
     temp_dir: Option<TempPath>,
 }
 
+#[cfg(any(test, feature = "fuzzing"))]
+impl Clone for TestConfig {
+    fn clone(&self) -> Self {
+        Self {
+            account_keypair: self.account_keypair.clone(),
+            consensus_keypair: self.consensus_keypair.clone(),
+            temp_dir: None,
+        }
+    }
+}
+
+#[cfg(not(any(test, feature = "fuzzing")))]
```

Comment:
do you know why this has to implement Clone for non-testing code?

Reply:
We should probably rename this to CloneForTemplate as done in other places in the code. The intent is to be agnostic to what is actually here so if other folks add features, they reason about what should be copied over and what should not. If we just instantiated a new "default" TestConfig during cloning of the config, it might break in weird ways.